### PR TITLE
January 10 2024 - Kris Stern contributor spotlight

### PR DESCRIPTION
**Pull request by kmartens27**
This pull request is to create the Contributor Spotlight page for @krisstern.

The publication date is currently set at January 10, 2024 so that we align with the general break that Jenkins is taking at the end of December. Additionally, this new date accounts for the decreased traffic & visibility that normally comes with the holidays & end of year.

### CODE REVIEW

File: src/pages/contributors/kris-stern.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+= Kris Stern
+:page-name: Kris Stern
+:page-linkedin: 
+:page-twitter: 
+:page-github: krisstern
+:page-email: 
+:page-image: avatar/kris-stern.jpg
```
Comment (krisstern) [inline src/pages/contributors/kris-stern.adoc:7]:
```suggestion
:page-image: avatar/kris-stern.png
```

**Review by krisstern — CHANGES_REQUESTED** [review]
I think we will need to set only one contributor's attribute `:page-featured:` to `true` at any one time, otherwise we will have two featured contributors like in this case.

File: src/pages/contributors/kris-stern.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
+= Kris Stern
+:page-name: Kris Stern
+:page-linkedin: 
+:page-twitter: 
+:page-github: krisstern
+:page-email: 
+:page-image: avatar/kris-stern.png
+:page-pronouns: They/Theirs
```
Comment (krisstern) [inline src/pages/contributors/kris-stern.adoc:8]:
```suggestion
:page-pronouns: They/Them/Theirs
```

**Review by krisstern — CHANGES_REQUESTED** [review]

**Review by kcsmalls — APPROVED** [review]

File: src/pages/contributors/kris-stern.adoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+:page-linkedin: 
+:page-twitter: 
+:page-github: krisstern
+:page-email: 
+:page-image: avatar/kris-stern.png
+:page-pronouns: They/Them/Theirs
+:page-location: Hong Kong SAR, China
+:page-firstcommit: 2021
+:page-datepublished: 2024-01-10
+:page-featured: false
```
Comment (krisstern) [inline src/pages/contributors/kris-stern.adoc:12]:
```suggestion
:page-featured: true
```

**Comment by krisstern** [comment]
Updated the date in the latest contributor feature to January 10, 2024.

**Review by krisstern — APPROVED** [review]
LGTM

**Review by alyssat — APPROVED** [review]
LGTM. I enjoyed reading our contributor stories :)

**Comment by krisstern** [comment]
@kmartens27 Checked that everything in the preview looks okay and the PR is ready to merge as scheduled today

**Comment by kmartens27** [comment]
Thanks very much @krisstern! Glad to have the final approval :)

**Pull request merged**